tests/test_parameters.py

```diff
@@ -647,4 +648,12 @@ def test_invalid_phosphoros_table(tab_value):
     with pytest.raises(RequestNotUnderstood):
         PhosphorosFiltersTable(tab_value)
     
+
+def test_structured_get_default_value():
+    value = {"c": [15, 5], "a": [1, 2], "b": ["foo", "bar"]}
+    expected = '{"a": [1, 2], "b": ["foo", "bar"], "c": [15, 5]}'
+    
+    stp = StructuredParameter(value)
+    
+    assert stp.get_default_value() == expected
```

Comment:
I'd add a test where the value is a `dict`

Reply:
Sorry, I don't understand what you mean. It's already a dict...

Reply:
true, I wanted to refer to a test for the new approach to populate `self.par_dic`, which checks if the value is a list or a dict, can we add a test for that ?

Reply:
The overall flow is tested in plugin, https://github.com/oda-hub/dispatcher-plugin-nb2workflow/blob/d0228f8f1e43da762785479b0ca16b924bf27561/tests/test_plugin.py#L554
It's not completely clear how to write a more localized test here. 

If we want to test only `InstrumentQueryBackEnd.set_args()` method, do you know how to prepare the `request` parameter as an instance of `flask.Request()`?

Reply:
what about a dummy instrument with a product that takes the new parameter as parameter a `StructuredParameter` ?

send a dummy request and then and check the content of the returned object ?